src/libhirte/test/common/network/get_address_test.c

```diff
@@ -0,0 +1,48 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+#include <assert.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "libhirte/common/network.h"
+
+#define bool_to_str(b) b ? "true" : "false"
```

Comment:
Nit: `bool_to_str` was only used in `string-utils_test` before. We could move it now to a central location, maybe [libhirte/common/string-utils](https://github.com/containers/hirte/blob/main/src/libhirte/common/string-util.h) or [libhirte/common/common)(https://github.com/containers/hirte/blob/main/src/libhirte/common/common.h)?